app/models/package.rb

```diff
@@ -571,6 +586,12 @@ def donor_condition_name
     donor_condition.try(:name_en) || item.try(:donor_condition).try(:name_en)
   end
 
+  def update_carts
+    cart_items.each do |cart_item|
```

Comment:
Again just FYI about ruby one-liner way:

cart_items.each(&:update_availability!)

You can keep this as well. Whatever you like :)

Reply:
cool stuff!